BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/file_collection/views/_plain_file_list_row.html.erb

```diff
@@ -41,7 +41,7 @@
     <% else %>
       <%= image_tag "/images/file_icon.png" %>
     <% end %>
-    <% if file.size > 0%>
+    <% if file.size > 0 && file.symbolic_type != :directory %>
```

Comment:
I see, but that cause another problem. The return is done too early. If we want to add data links for things that are not files, e.g. subdirectories that match a FileCollection model, we can't.